test/commands/user_track/generate_concepts_status_test.rb

```diff
@@ -0,0 +1,177 @@
+require "test_helper"
+
+class UserTrack::GenerateConceptsStatusTest < ActiveSupport::TestCase
+  def setup_user_track
+    track = create :track
+    user_track = create :user_track, track: track
+
+    [track, user_track]
+  end
```

Comment:
I think for this method, I'd just have `create :user_track` and then use `user_track.track` or `track = user_track.track`. I'm not sure this one adding much.

Reply:
Meh - actually. I've changed my mind on this reading it below. It's good and makes the tests really clear. Ignore my comment 🙂